webpack/components/TypeAhead/pf4Search/TypeAheadInput.js

```diff
@@ -31,19 +31,19 @@ const TypeAheadInput = ({
         onFocus={onInputFocus}
         aria-label="text input for search"
         onChange={onChangeWrapper}
-        type="search"
-        iconVariant={autoSearchEnabled && 'search'}
+        type="text"
+        iconVariant={(autoSearchEnabled && !isTextInput) && 'search'}
```

Comment:
This can produce a case where false is returned. 
(Which may not cause any harm, but not the best standard).
We should likely be explicit and use a ternary operator
example `autoSearchEnabled && !isTextInput ? 'search' : undefined`